ktor-utils/jvm/src/io/ktor/util/CryptoJvm.kt

```diff
@@ -55,7 +67,7 @@ private inline class DigestImpl(val delegate: MessageDigest) : Digest {
     ReplaceWith("s.toByteArray(Charsets.UTF_8)"),
     level = DeprecationLevel.ERROR
 )
-fun raw(s: String) = s.toByteArray(Charsets.UTF_8)
+fun raw(s: String): ByteArray = s.toByteArray(Charsets.UTF_8)
```

Comment:
Could it be private or should we rename the parameter and add the doc?

Reply:
It is already deprecated with error, going to be dropped